internal/app/carrier/repository.go

```diff
@@ -0,0 +1,37 @@
+package carrier
+
+import (
+	"context"
+	"fmt"
+	"parcel-service/internal/app/model"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/lib/pq"
+)
+
+// sql query and error
+const (
+	errUniqueViolation = pq.ErrorCode("23505")
+	insertCarrierQuery = `INSERT INTO carrier_request (carrier_id, parcel_id) VALUES ($1, $2)`
```

Comment:
stats is required. how status will inserted at there?

Reply:
@shuza  status value is default 1 in DB. No need to insert default value from code,  `status INT NOT NULL DEFAULT 1`